README.md

```diff
@@ -10,117 +10,140 @@
 
 ## What is Changelog CI?
 
-Changelog CI is a GitHub Action that generates changelog,
-Then the changelog is prepended to ``CHANGELOG.md`` file and **committed** 
-and/or **commented** to the release Pull request.
+Changelog CI is a GitHub Action that enables a project to utilize an
+automatically generated changelog.
+
+This action will:
+
+* **Prepend** each update to the `CHANGELOG.md` file and
+
+* **Commit** the modified `CHANGELOG.md` file.
+
+* (optionally) **Write** the updates as a comment to the pull request of
+  the corresponding release version.
```

Comment:
These are and/or option, So, you can have both or just one of them (Commit or Comment)

Reply:
Good catch, will resolve this in the next commit.